config/manifests/bases/camel-k.clusterserviceversion.yaml

```diff
@@ -22,15 +22,15 @@ metadata:
     capabilities: Full Lifecycle
     categories: Integration & Delivery
     certified: "false"
-    containerImage: docker.io/apache/camel-k:1.5.0-SNAPSHOT
+    containerImage: docker.io/apache/camel-k:1.6.0-SNAPSHOT
     createdAt: 2021-05-03T07:48:00Z
     description: Apache Camel K is a lightweight integration platform, born on Kubernetes, with serverless superpowers.
     operators.operatorframework.io/builder: operator-sdk-v1.3.0
     operators.operatorframework.io/internal-objects: '["builds.camel.apache.org","integrationkits.camel.apache.org","camelcatalogs.camel.apache.org"]'
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
     repository: https://github.com/apache/camel-k
     support: Camel
-  name: camel-k.v1.5.0
+  name: camel-k.v1.6.0-snapshot
```

Comment:
If memory serves, you cannot suffix this with -snapshot as operator-sdk considers it not to conform with semver syntax.